src/template.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.6.1/jquery.min.js"></script>
+
+    <title>%= htmlWebpackPlugin.options.title %</title>
+  </head>
+  <body>
+    <main class="container">
+      <div>
+        <h1 for="cars">ToDo List:</h1>
+        <img id="recyclImg" src="./assets/refresh.png" alt="refresh todo list" />
+      </div>
+      <div>
+        <input id="userInput" type="text" placeholder="Add to the list" required autofocus />
+        <img id="addBtn" alt="add new task" src="./assets/enter.png" />
+      </div>
+
+      <ul id="tasksList"></ul>
+
+      <button id="clearAllBtn" type="button">Clear all Completed</button>
+    </main>
+  </body>
+</html>
+
+<!-- <header><h1>ToDo List</h1></header>
+<main>
+  <ul class="container">
+    <li>
+      <h2>Add Task:</h2>
+      <img id="recyclImg" src="./assets/refresh.png" alt="refresh todo list" />
+    </li>
+    <li>
+      <input id="userInput" type="text" placeholder="Add to the list" required autofocus />
+      <img id="addBtn" alt="add new task" src="./assets/enter.png" />
+    </li>
```

Comment:
Using best practices, I think it is good you remove unused codes in your work. Unused codes left as comments makes your code messy and difficult to read